openedx/core/djangoapps/user_authn/views/login_form.py

```diff
@@ -53,8 +54,11 @@ def login_and_registration_form(request, initial_mode="login"):
     """
     # Determine the URL to redirect to following login/registration/third_party_auth
     redirect_to = get_next_url_for_login_page(request)
+
     # If we're already logged in, redirect to the dashboard
-    if request.user.is_authenticated:
+    # Note: we check for the existence of login-related cookies in addition to is_authenticated
+    #   since session cookies are auto-updated more frequently than other cookies.
```

Comment:
When/how are session cookies auto-updated? What will be the consequence of no longer letting the session authentication live? More frequent requests for login? Would these be reasonable requests for login? Would there be any need or possibility of refreshing the other cookies while in this state to match is_authenticated?